### ISSUE TITLE
Version 3.2 ChangeLog

### ISSUE THREAD

**Issue by anthrotype** [fonttools/fonttools#702]
I would like to cut a new 3.2.0 release soon.
This is the draft of the change log. Feel free to comment or add new entries if you wish.

Also, it would be nice if the future pull requests would update the `NEWS` file with a summary of the changes so this release task becomes a bit less tedious. 😁 

WDYT? 
---
- [varLib] Improve support for OpenType 1.8 Variable Fonts:
  - Implement GDEF's VariationStore
  - Implement HVAR/VVAR tables
  - Partial support for loading MutatorMath .designspace files with `varLib.designspace` module
  - Add `varLib.models` with Variation fonts interpolation models
  - Implement GSUB/GPOS FeatureVariations
  - Initial support for interpolating and merging OpenType Layout tables (see `varLib.interpolate_layout` and `varLib.merger`)
- [API change] Change version to be an integer instead of a float in XML output for GSUB, GPOS, GDEF, MATH, BASE, JSTF, HVAR, VVAR, and feat tables. Scripts that set the Version for those to 1.0 or other float values also need fixing. A warning is emitted when code or XML needs fix. (TODO: fix all the remaining ones that do not use otConverters.Version as well)
- several bug fixes to the `cffLib` module, contributed by Adobe's @readroberts
- The XML output for CFF table now has a 'major' and 'minor' elements for specifying whether it's version 1.0 or 2.0
- [setup.py] remove undocumented/deprecated 'extra_path' Distutils argument. This means that we no longer create a "FontTools" subfolder in site-packages containing the actual `fontTools` package, as well as the standalone `xmlWriter` and `sstruct` modules. The latter modules are also deprecated, and scheduled for removal in upcoming releases. Please change your import statements to point to `from fontTools.misc import xmlWriter` and `from fontTools.misc import sstruct`.
- [versioning] Use `versioneer` instead of `setuptools_scm` to dynamically load version info from a git checkout at import time.

**Comment by anthrotype** [comment]
Also, please let me know if there's something unfinished that you'd like to be part of v3.2.0 and whether I should wait before tagging a release until that is completed. Thank you.

**Comment by behdad** [comment]
Wait a few hours for me to finish binary interpolation please.

**Comment by anthrotype** [comment]
Of course, no rush. We can wait as long as you need. That's why I opened this. ;)

**Comment by anthrotype** [comment]
@behdad sorry to ping you, I know you're busy. Any news on the binary interpolation front?
Do you think we could tag 3.2.0 as it is now, and push the rest to a later release?

**Comment by anthrotype** [comment]
do we have an estimated time frame when we are able to tag 3.2.0?

There are several patches that could potentially go to a 3.1.3 release, but are being held off by other incomplete stuff which has been pushed to master, perhaps prematurely.

IMO we should try to push/merge into master only code that is deemed "releaseable" at any point in time. Things which are not finished should live in separate branches (rebased from time to time), and should be merged once they are completed.

Otherwise we find ourselves into the the same stalemate which makes us release only once a year, and forces users to use the development version (unstable by definition), and blah blah.

**Comment by behdad** [comment]
Hey. Yes sorry forgot to let you know. There particular work I was talking about was finished a week ago.

**Comment by anthrotype** [comment]
thanks. So I can update the changelog and then pull the trigger?

**Comment by behdad** [comment]
Please!
